packages/material-ui-icons/package.json

```
@@ -64,5 +64,8 @@
   },
   "engines": {
     "node": ">=8.0.0"
+  },
+  "exports": {
+    "./*": "./*.js"
```

The main should likely remain included. For backwards compat it can be useful to export the package.json. Perhaps include the ES versions as well:

```suggestion
    ".": "./index.js",
    "./package.json": "./package.json",
    "./*": {
      "require": "./*.js",
      "import": "./es/*.js"
```